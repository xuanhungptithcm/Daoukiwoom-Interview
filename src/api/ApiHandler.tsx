const ApiHandler = (): null => {
  // to call API when first loading page
  return null;
};

export default ApiHandler;
