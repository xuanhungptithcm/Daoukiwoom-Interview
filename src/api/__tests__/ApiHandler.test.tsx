import { shallow } from 'enzyme';
import React from 'react';
import ApiHandler from '../ApiHandler';

describe('ApiHandler', () => {
  test('renders correct', () => {
    expect(shallow(<ApiHandler />)).toMatchSnapshot();
  });
});