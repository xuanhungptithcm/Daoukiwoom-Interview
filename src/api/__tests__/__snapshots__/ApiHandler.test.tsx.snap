// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApiHandler renders correct 1`] = `""`;
